pt-BR/mail/chrome/messenger/messengercompose/askSendFormat.dtd

```diff
@@ -4,22 +4,22 @@
 <!ENTITY recipient.label "Esta mensagem contém formatação. Alguns dos destinatários podem não conseguir abrí-la corretamente.">
 
 <!ENTITY convertibleDefault.label "">
-<!ENTITY convertibleYes.label "Your message can be converted to plain text without losing information.">
-<!ENTITY convertibleAltering.label "Your message can be converted to plain text without losing important information. However, the plain text version might look different from what you saw in the composer.">
-<!ENTITY convertibleNo.label "However, you used formatting (e.g. colors) that will not be converted to plain text.">
+<!ENTITY convertibleYes.label "Sua mensagem pode ser convertida para texto sem formatação sem que haja perda de informações.">
+<!ENTITY convertibleAltering.label "Sua mensagem pode ser convertida para texto sem formatação sem que haja perda de informações importantes. Entretanto, a versão sem formatação pode ter um visual diferente do que foi exibido ao criar a mensagem.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY convertibleAltering.label "Sua mensagem pode ser convertida para texto sem formatação sem que haja perda de informações importantes. Contudo, a versão sem formatação pode ter uma apresentação diferente do que foi exibido ao criar a mensagem.">